projects/sidechain-connector-service-typescript/src/test/ethereum-connector.spec.ts

```diff
@@ -0,0 +1,77 @@
+const {Assertion, expect} = require("chai");
+const Web3 = require("web3");
+const ganache = require("ganache-cli");
+const path = require("path");
+const fs = require("fs");
+const solc = require("solc");
+
+import EthereumConnector from "../ethereum-connector";
+
+const SIMPLE_STORAGE_SOLIDITY_CONTRACT = `
+pragma solidity ^0.4.17;
```

Comment:
Thinking out-loud: eventually, we might find ourselves calling contracts which were compiled with a different solidity compiler (which can result in a different and incompatible API). Should we support it (outside the tests, that is)? E.g., by adding a compiler version to the message?

Reply:
Eventually yes. it's not a requirement at the moment